docs/source/en/api/pipelines/mochi.md

```diff
@@ -25,6 +25,50 @@ Make sure to check out the Schedulers [guide](../../using-diffusers/schedulers.m
 
```

Comment:
(One line above this) Only FlowMatchEulerDiscreteScheduler has invert_sigmas, so anything else wouldn't as of now as I understand it

Reply:
Cc: @hlky here too. @DN6 do we wanna remove this bit? I think we need to remove it from all flow pipelines.